build/tools/Automation/Models/YamlConstants.cs

```diff
@@ -0,0 +1,20 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+using System.Collections.Generic;
+
+namespace Microsoft.Oryx.Automation.Models
+{
+    /// <Summary>
+    /// This is used to deserialize Constants.ConstantsYaml file
+    /// </Summary>
+    public class YamlConstants
```

Comment:
I would avoid having the `Constants` suffix on this class name if the class isn't encapsulating static string constants (I know this is focused on the `constants.yaml` file, so maybe a better name is `ConstantsYamlFile`)